NEWS.md

```diff
@@ -105,7 +108,7 @@
 ### Broom
   
 - Implemented `tidy()`, `augment()`, and `glance()` from tidyverse/broom for
-  `ml_model_generalized_linear_regression()` and `ml_model_linear_regression()`
+  `ml_model_generalized_linear_regression` and `ml_model_linear_regression`
```

Comment:
Can you restore the parentheses here, please? (it's preferred to use those to make it more apparent that we're talking about functions as opposed to other kinds of objects)